locales/fa/fa.json

```diff
@@ -36,8 +36,7 @@
     "All rights reserved.": "کلیه حقوق محفوظ است.",
     "Already registered?": "قبلا ثبت نام کرده اید؟",
     "American Samoa": "ساموآی آمریکا",
-    "An error occured while uploading the file.": "هنگام بارگذاری فایل خطایی رخ داده است.",
```

Comment:
Do not remove lines from the file. Yes, the line contains a grammatical error, BUT it occurs in Laravel Nova 3.